### ISSUE TITLE
Py35 support broken again.

### ISSUE THREAD

**Issue by hugonicodemos** [HewlettPackard/python-ilorest-library#8]
After the commit https://github.com/HewlettPackard/python-ilorest-library/commit/5966b26767137ac3e04bd4792ed6f8c165d367e2 the supports to py35 breaks again. I recommend using library six instead of httplib and StringIO libraries, as these are not supported in py35.

**Comment by Yergidy** [comment]
See if this fixes your issue.

**Comment by hugonicodemos** [comment]
Hey @Yergidy, thanks for the commit and it fixes, but now we need a new release in pipy with your commit for our CI works properly.

**Comment by ricardoas** [comment]
Is it possible to enable py35 at travis evaluation?

**Comment by Yergidy** [comment]
@hugonicodemos We will be releasing a new pypi build when the next version releases soon.

**Comment by ricardoas** [comment]
@Yergidy I tried to use the repo version and it continued breaking now with:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/py3_test_ilo/lib/python3.5/site-packages/python_ilorest_library-2.0.0-py3.5.egg/redfish/rest/v1.py", line 1368, in redfish_client
  File "/tmp/py3_test_ilo/lib/python3.5/site-packages/python_ilorest_library-2.0.0-py3.5.egg/redfish/rest/v1.py", line 1439, in get_client_instance
  File "/tmp/py3_test_ilo/lib/python3.5/site-packages/python_ilorest_library-2.0.0-py3.5.egg/redfish/rest/v1.py", line 1077, in __init__
  File "/tmp/py3_test_ilo/lib/python3.5/site-packages/python_ilorest_library-2.0.0-py3.5.egg/redfish/rest/v1.py", line 497, in __init__
AttributeError: module 'urlparse2' has no attribute 'urlparse'
```

I noticed guys at dmtf [solved this](https://github.com/DMTF/python-redfish-library/blob/master/src/redfish/rest/v1.py) using the `six` lib. Is it possible to add this too?

**Comment by Yergidy** [comment]
@ricardoas Yes. This will be added with our next full release soon.

**Comment by Yergidy** [comment]
We have released version 2.1 which includes full python 3 compatibility. A pip version will follow in the coming days.

**Comment by hugonicodemos** [comment]
Thanks, @Yergidy

**Comment by Yergidy** [comment]
pypi versions are now available for version 2.1
